layers.go

```diff
@@ -491,7 +491,7 @@ func (r *layerStore) saveMounts() error {
 	return r.loadMounts()
 }
 
-func (s *store) newLayerStore(rundir string, layerdir string, driver drivers.Driver) (LayerStore, error) {
+func (s *store) newLayerStore(rundir string, layerdir string, driver drivers.Driver, withMountsLock bool) (LayerStore, error) {
```

Comment:
Please add a comment explaining the implications of `withMountsLock`. A comment at the definition of `layerStore` would be great as well, so that IDE's and godef will help understanding it.

Reply:
Added comment as suggested :+1: